docs/user-defined-redirects.rst

```diff
@@ -59,9 +68,14 @@ You would set the following configuration::
     From URL: /example.html
     To URL: /examples/intro.html
 
-Note that the ``/`` at the start doesn't count the ``/en/latest``, 
+Note that the ``/`` at the start doesn't count the ``/en/latest``,
 but just the user-controlled section of the URL.
 
+.. tip::
+
+   *Page Redirects* can redirect URLs **outside** Read the Docs platform.
```

Comment:
:+1:

Reply:
Maybe is worth mentioning that is enough to put an absolute url to archive this?